ipyparallel/controller/hub.py

```diff
@@ -833,15 +833,26 @@ def mia_task_request(self, idents, msg):
 
     #--------------------- IOPub Traffic ------------------------------
 
-    def save_iopub_message(self, topics, msg):
-        """save an iopub message into the db"""
-        # print (topics)
+    def monitor_iopub_message(self, topics, msg):
+        '''intercept iopub traffic so events can be acted upon'''
```

Comment:
I'm not really a fan of this method: intercepting the messages on iopub doesn't seem like the right solution, so I'm open to better ways of doing this